Sources/Controllers/Settings/NavigationSettings/OAVehicleParametersSettingsViewController.mm

```diff
@@ -137,13 +137,23 @@ - (void)setupTableHeaderView
         topImageDivider.backgroundColor = UIColorFromRGB(color_tint_gray);
         
         UIImage *image = [UIImage imageNamed:[self getParameterImage:_vehicleParameter[@"name"]]];
-        CGFloat aspectRatio = MIN(DeviceScreenWidth, DeviceScreenHeight) / image.size.width;
-        UIImageView *imageView = [[UIImageView alloc] initWithFrame:CGRectMake(0., 0., DeviceScreenWidth, image.size.height * aspectRatio)];
-        imageView.image = image;
-        imageView.contentMode = UIViewContentModeScaleAspectFit;
-        
-        UIView *imageBackgroundView = [[UIView alloc] initWithFrame:CGRectMake(0., 0.5, DeviceScreenWidth, imageView.frame.size.height)];
-        imageBackgroundView.backgroundColor = UIColor.whiteColor;
+        UIImageView *imageView = nil;
```

Comment:
It looks like you should remove the extra top space in cases where there is no image